src/test/groovy/com/stormpath/tck/util/TestAccount.groovy

```diff
@@ -16,23 +16,31 @@ class TestAccount {
     final String username = "foo-" + randomUUID
     String href
 
-    def TestAccount() {
-        def account = [email: email,
-            password: password,
-            givenName: givenName,
-            surname: surname,
-            middleName: middleName,
-            username: username]
+    public static TestAccount create() {
```

Comment:
Ah, so you can use the test data for testing registration? I think static method makes sense so initialization is easier in other files. Sure, I would just rename `TestAccount.create()` to `TestAccount.registerOnServer()` or something so it's clear what "create" means vs. "new".